Source/WebGPU/WGSL/Parser.cpp

```diff
@@ -46,26 +46,15 @@ namespace WGSL {
 
 #define RETURN_NODE(type, ...) \
     do { \
-        AST::type astNodeResult(CURRENT_SOURCE_SPAN(), __VA_ARGS__); \
-        return { WTFMove(astNodeResult) }; \
-    } while (false)
-
-// Passing 0 arguments beyond the type to RETURN_NODE is invalid because of a stupid limitation of the C preprocessor
-#define RETURN_NODE_NO_ARGS(type) \
-    do { \
-        AST::type astNodeResult(CURRENT_SOURCE_SPAN()); \
+        AST::type astNodeResult(CURRENT_SOURCE_SPAN() __VA_OPT__(,) __VA_ARGS__); /* NOLINT */ \
```

Comment:
Ha, TIL about `__VA_OPT__`. I think I used `##` somewhere, should probably remove that